README.md

```diff
@@ -476,6 +476,10 @@ client.process_template template
 
 ## Upgrading
 
+### past version 3.0
+
+Deprecated KubeException has been dropped, use `Kubeclient::HttpException` instead.
+
```

Comment:
Should be `Kubeclient::HttpError`.

Reply:
see https://github.com/abonas/kubeclient/pull/288